lib/db_ido/dbevents.cpp

```diff
@@ -879,9 +879,10 @@ void DbEvents::AddNotificationHistory(const Notification::Ptr& notification, con
 		fields1->Set("endpoint_object_id", endpoint);
 
 	query1.Fields = fields1;
-	DbObject::OnQuery(query1);
+	query1.RunAlone = true;
 
 	std::vector<DbQuery> queries;
+	queries.emplace_back(std::move(query1));
```

Comment:
You are moving `query1` here, then you access it down [here](https://github.com/Icinga/icinga2/blob/master/lib/db_ido/dbevents.cpp#L898).